### DISCUSSION TITLE
how to make it shadowy

### DISCUSSION THREAD

**Discussion by iamzhongjian** [google/filament#6427] [Q&A]
[Android]Hi, I have a 3D model in the center of the sky box. I want it to have shadows on the ground. Is there any way? Do I need to add ground?how to add ground plane?

![image](https://user-images.githubusercontent.com/57252697/210697390-7842e89e-0af8-43a4-b15b-5470b1d1398e.png)

**Comment by bejado** [comment]
You can add a ground plane with a material that has `shadowMultiplier` turned on. See [gltf_viewer.cpp](https://github.com/google/filament/blob/c7c184d952c4f7d874a0d382a5c8b5c8aada96a9/samples/gltf_viewer.cpp#L287-L374) and [groundShadow.mat](https://github.com/google/filament/blob/main/samples/materials/groundShadow.mat).
